### PR TITLE
Fixes #1072 Current selected reminder not saved when editing a reminder 

### LINKED ISSUE #1072: Current selected reminder not saved when editing a reminder
- Add someone
- Add a reminder, and set it to eg. "Remind me about this every 2 week" 
- Save
- Update the reminder
=> the saved reminder time (every 2 weeks) is not selected.

### PR DESCRIPTION
**Pull request by turtles2**
Fixes #1072 Current selected reminder not saved when editing a reminder

### CODE REVIEW

**Comment by turtles2** [comment]
@djaiss or @asbiin Can we get a review on this PR?

File: resources/views/people/reminders/form.blade.php [hunk 1]
```diff
@@ -50,15 +57,33 @@
...
                            :disabled="reminders_frequency == 'once'">
 
                     <select name="frequency_type" :disabled="reminders_frequency == 'once'">
-                        <option value="week">{{ trans('people.reminders_type_week') }}</option>
-                        <option value="month">{{ trans('people.reminders_type_month') }}</option>
-                        <option value="year">{{ trans('people.reminders_type_year') }}</option>
+                        @if($update_or_add == 'edit' && $reminder->frequency_number != 'one_time')
+
+                            @if($reminder->frequency_type == 'week')
+                                <option selected value="week">{{ trans('people.reminders_type_week') }}</option>
```
Comment (asbiin) [inline resources/views/people/reminders/form.blade.php]: You can do once:
```htm
<option value="week" {{ $update_or_add == 'edit' && $reminder->frequency_type == 'week' ? 'selected : '' }}>{{ trans('people.reminders_type_week') }}</option>
<option value="month" {{ $update_or_add == 'edit' && $reminder->frequency_type == 'month' ? 'selected : '' }}>{{ trans('people.reminders_type_month') }}</option>
<option value="year" {{ $update_or_add == 'edit' && $reminder->frequency_type == 'year' ? 'selected : '' }}>{{ trans('people.reminders_type_year') }}</option>
```

File: resources/views/people/reminders/form.blade.php [hunk 2]
```diff
@@ -50,15 +57,15 @@
...
+                           value="{{$reminder->frequency_number ?? '1'}}"
                            min="1"
                            max="115"
                            :disabled="reminders_frequency == 'once'">
 
                     <select name="frequency_type" :disabled="reminders_frequency == 'once'">
-                        <option value="week">{{ trans('people.reminders_type_week') }}</option>
-                        <option value="month">{{ trans('people.reminders_type_month') }}</option>
-                        <option value="year">{{ trans('people.reminders_type_year') }}</option>
+                        <option value="week" {{ $update_or_add == 'edit' && $reminder->frequency_type == 'week' ? 'selected : '' }}>{{ trans('people.reminders_type_week') }}</option>
```
Comment (djaiss) [inline resources/views/people/reminders/form.blade.php]: This leads to an error. A `'` is missing after `selected`.
```
{{ $update_or_add == 'edit' && $reminder->frequency_type == 'week' ? 'selected HERE : '' }}
```

The same error appears on the 2 lines below.

**Comment by turtles2** [comment]
Sorry about that I have been making quick edits over GitHub so I have not
been able to run the changes.

**Comment by turtles2** [comment]
@djaiss Hope you don't mind but I added an account to the heroku instance so that I could test my changes. I have fixed it and it appears to be working.

**Comment by djaiss** [comment]
@turtles2 thanks!

Unfortunately it doesn't work.
* Create a one-time reminder
* Try edit it. First of all it has become a recurrent reminder, and we can't change the value back.

This is due because of `:onload="reminders_frequency = 'recurrent'"` that you've added.

I don't think we should spend more time on this. I planned to rewrite this page anyway and make it much better.

**Pull request closed without merging**